src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
Just to summarize again the options as I see them:

- Implicitly insert `/usr/local` unless some heuristic is met, e.g. don't insert it if:
  - `rospack` can be found outside of `/usr` or
  - `ROS_NOT_DEBIAN` is set in the environment (this could be set in `/opt/ros/kinetic/setup.*sh`)
- Don't change `rospkg`, but just install the normal setup files in an appropriate location, have users source them before using (violates 9.9)
- Have a different name for the packages.ros.org version of `rospkg` (not sure this is a tractable solution)